### ISSUE TITLE
Executing letsencrypt outside of git repo (when globally installed) fails looking for EULA

### ISSUE THREAD

**Issue by pavgup** [letsencrypt/lets-encrypt-preview#278]
I think the EULA just has to be included in the package, but I've not had a moment to try it.  Sorry for the spam!  Here's the error -- nothing crazy (and this occurs in every flavor of linux I poked at this morning):
```
(venv)root@vpn:~# letsencrypt
Traceback (most recent call last):
  File "/root/lets-encrypt-preview/venv/bin/letsencrypt", line 9, in <module>
    load_entry_point('letsencrypt==0.1', 'console_scripts', 'letsencrypt')()
  File "/root/lets-encrypt-preview/venv/local/lib/python2.7/site-packages/letsencrypt-0.1-py2.7.egg/letsencrypt/scripts/main.py", line 137, in main
    display_eula()
  File "/root/lets-encrypt-preview/venv/local/lib/python2.7/site-packages/letsencrypt-0.1-py2.7.egg/letsencrypt/scripts/main.py", line 194, in display_eula
    with open("EULA") as eula_file:
IOError: [Errno 2] No such file or directory: 'EULA'
(venv)root@vpn:~#
```

**Comment by jdkasten** [comment]
Thanks for reporting this @pavgup.  

It should be using pkg_resources, however this code will soon be replaced by...
https://github.com/letsencrypt/acme-spec/pull/69

**Comment by pavgup** [comment]
Totally my pleasure.  You guys rock.  I actually stumbled in to the explicit agreement changes, but I guessed the EULA was largely standing in as a disclaimer against pre-release software usage.  I should say I love the idea that the client would control the flow of acceptances -- it allows a discerning person to reject requirements that might be too onerous in the open rather than stumbling into onerous requirements in the moment.
